### ISSUE TITLE
KeyError: 'bytes'

### ISSUE THREAD

**Issue by jacovdbergh** [Johannes11833/rclone_python#50]
Latest version that uses --stats instead of --progress introduced the following bug for me

Running  `rclone.copy(f"remote:some-bucket/some-folder", f"/opt/task/some-folder", ["--include=*.laz", "--include=*.las"])`

```
  File "/usr/local/lib/python3.11/dist-packages/rclone_python/rclone.py", line 184, in copy
    _rclone_transfer_operation(
  File "/usr/local/lib/python3.11/dist-packages/rclone_python/rclone.py", line 24, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/rclone_python/rclone.py", line 665, in _rclone_transfer_operation
    process = utils.rclone_progress(
              ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/rclone_python/utils.py", line 91, in rclone_progress
    valid, update_dict = extract_rclone_progress(line)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/rclone_python/utils.py", line 139, in extract_rclone_progress
    "sent": t["bytes"],
            ~^^^^^^^^^
KeyError: 'bytes'
Python script exited with code 1
```

In v0.1.15 this does not happen.

**Comment by Johannes11833** [comment]
That issue should be fixed now in [v0.1.17](https://github.com/Johannes11833/rclone_python/releases/tag/v0.1.17). Sometimes the progress information of a specific task is not available right from the starts

**Comment by jacovdbergh** [comment]
Awesome, thanks for the quick response!
